src/VodafoneChannel.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace NotificationChannels\Vodafone;
+
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification;
+
+class VodafoneChannel
+{
+    /**
+     * @var string Vodafone's API endpoint
+     */
+    protected $endpoint = 'https://www.smsertech.com/apisend';
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     *
+     * @throws \NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        $message = $notification->toVodafone($notifiable, $notification);
```

Comment:
I'd double check before this that the method exists, and then after to make sure it returned a valid object.